src/transformers/image_utils.py

```diff
@@ -220,7 +315,7 @@ def normalize(self, image, mean, std):
         else:
             return (image - mean) / std
 
-    def resize(self, image, size, resample=PIL.Image.BILINEAR, default_to_square=True, max_size=None):
+    def resize(self, image, size, resample=None, default_to_square=True, max_size=None):
```

Comment:
Moved to be within the function, rather than in the signature to avoid issues when importing and PIL isn't installed